pages/pipelines/command_step.md.erb

```diff
@@ -2,26 +2,142 @@
+      <em>Example:</em> <code>"master stable/*"</code>
+    </td>
+  </tr>
+  <tr>
+    <td><code>env</code></td>
+    <td>
+      A list of <a href="/docs/builds/environment-variables">environment variables</a> for this step. <br>
+      <em>Example:</em> <code>FOO: "bar"</code>
+    </td>
+  </tr>
+  <tr>
+    <td><code>agents</code></td>
+    <td>
+      A list of <a href="/docs/agent/cli-meta-data">meta-data</a> key value pairs to <a href="/docs/agent/cli-start#agent-targeting">target specific agents</a> for this step. <br> 
+      <em>Example:</em> <code>npm: "true"</code>
+    </td>
+  </tr>
+  <tr>
+    <td><code>artifact_paths</code></td>
+    <td>
+      The <a href="/docs/rest-api/artifacts#glob-path-pattern">glob path</a> or paths where <a href="/docs/agent/cli-artifact">artifacts</a> from this step will be uploaded. This can be a single line of paths separated by semicolons, or a yaml list.<br>
+      <em>Example:</em> <code>"logs/**/*;coverage/**/*"</code>
+    </td>
+  </tr>
+  <tr>
+    <td><code>parallelism</code></td>
+    <td>
+      The number of <a href="/docs/builds/parallelizing-builds#job-parallelism">parallel jobs</a> that will be created based on this step. <br>
+      <em>Example:</em> <code>3</code>
+    </td>
+  </tr>
+  <tr>
+    <td><code>concurrency</code></td>
+    <td>
+      The <a href="/docs/builds/controlling-concurrency#concurrency-limits">maximum number of jobs</a> created from this step that are allowed to run at the same time. If you use a concurrency limit, you must also define a label for it in the concurrency_group attribute.<br>
+      <em>Example:</em> <code>3</code>
+    </td>
+  </tr>
+  <tr>
+    <td><code>concurrency_group</code></td>
+    <td>
+      A unique name for the concurrency group that you are creating with the <code>concurrency</code> attribute. This can share a name with a <a href="/docs/builds/controlling-concurrency#concurrency-groups">concurrency group</a> set in another step or pipeline, if they're accessing a shared resource. <br>
+      <em>Example:</em> <code>"serial_tests"</code>
+    </td>
+  </tr>
+  <tr>
+    <td><code>timeout_in_minutes</code></td>
+    <td>
+      The number of minutes a job created from this step is allowed to run. If the job does not finish within this limit, it will be automatically cancelled and the build will fail. <br>
+      <em>Example:</em> <code>60</code>
+    </td>
+  </tr>
+</table>
 
 ```yml
-- command: "tests.sh"
-  name: "\:hammer\: Tests"
-  branches: "master"
-  env:
-    FOO: "bar"
-  agents:
-    npm: "true"
-    queue: "tests"
-  artifact_paths: "logs/**/*;coverage/**/*"
-  parallelism: 1
-  concurrency: 1
-  concurrency_group: "serial_tests"
-  timeout_in_minutes: 60
-```
\ No newline at end of file
+steps:
+  - name: "\:hammer\: Tests"
+    command: "npm install && tests.sh"
+    branches: "master"
+    env:
+      FOO: "bar"
+    agents:
+      npm: "true"
+      queue: "tests"
+    plugins:
```

Comment:
Perhaps `plugins` should be removed from this example since it's not officially out yet?